drivers/builtin/include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -92,10 +92,7 @@
     defined(MBEDTLS_ENTROPY_C) || \
     defined(MBEDTLS_PK_C) || \
     defined(MBEDTLS_PKCS12_C) || \
-    defined(MBEDTLS_RSA_C) || \
-    defined(MBEDTLS_SSL_TLS_C) || \
-    defined(MBEDTLS_X509_USE_C) || \
-    defined(MBEDTLS_X509_CREATE_C)
```

Comment:
The commit says
`This commit removes MBEDTLS_SSL_TLS_C, MBEDTLS_X509_USE_C and
MBEDTLS_X509_CREATE_C from TF-PSA-Crypto to Mbed TLS where they more
appropriately belong.`

but I cannot see them being added somewhere in https://github.com/Mbed-TLS/mbedtls/pull/9915. Am I missing something?

Reply:
See https://github.com/Mbed-TLS/mbedtls/pull/9915#issuecomment-2630361279 explaining why we can actually just remove `defined(MBEDTLS_X509_USE_C) || defined(MBEDTLS_X509_CREATE_C)` here. Regarding MBEDTLS_SSL_TLS_C, it depends on MBEDTLS_MD_C as stated in mbedtls_config.h thus it can be just removed as well.

Reply:
Those commits were dropped from MbedTLS as I believe x509 doesn't have a dependency on them afterall, this commit message needs to be changed to reflect the real reason now :)

Reply:
We have discussed the various issues in several places, I completely understand it is not easy to follow.